internal/provider/provider.go

```diff
@@ -191,6 +191,12 @@ func Provider() *schema.Provider {
 				Set:           schema.HashString,
 			},
 			"assume_role": assumeRoleSchema(),
+			"custom_ca_bundle": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: "File containing custom root and intermediate certificates. " +
+					"Can also be configured using the `AWS_CA_BUNDLE` environment variable.",
```

Comment:
This setting also exists in the shared config file (`ca_bundle`), but reading it from the config file isn't currently supported by the AWS SDK (aws/aws-sdk-go-v2#1589). Maybe we should explicitly reference this, since we do support many settings from the file

Reply:
Done